.github/workflows/build-and-publish.yml

```diff
@@ -19,3 +23,9 @@ jobs:
         AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
         SOURCE_DIR: 'generated' 
         DEST_DIR: 'trackerblocking/config/v1'
+    - uses: EndBug/add-and-commit@v7
```

Comment:
I'm a bit of a beginner at reading these files (we should probably have an additional reviewer that has done more of it like @GioSensation), are we actually calling `node index.js` or `npm start` or similar here? Also if the unit tests just added fail, would this break before uploading to s3?

Reply:
This action only runs _after_ merge so tests should be passing at this stage. Also `line 16` is what runs `index.js` via [this file](https://github.com/duckduckgo/privacy-configuration/blob/main/action.yml)

Reply:
I think ideally it should also run tests just in case a broken merge gets put into main?